apps/funding-platform/src/entry-server/entry-server.ts

```diff
@@ -0,0 +1,110 @@
+import { AccessTokenService } from "../access-token";
+import express, { NextFunction, Request, Response } from "express";
+import { CreateRequest, RequestService } from "../request";
+
+const app = express();
+
+const THIRTY_MINUTES = 30;
+const MAX_HOPR = 40;
+
+const tokenIsValid =
+  (accessTokenService: AccessTokenService, requestService: RequestService) =>
+  async (req: Request, res: Response, next: NextFunction) => {
+    const accessTokenHash = req.headers["x-access-token"] as string;
+    if (!accessTokenHash) return res.status(400).json("Missing Access Token");
+    const dbToken = await accessTokenService.getAccessToken(accessTokenHash);
+    if (!dbToken) return res.status(404).json("Access Token does not exist");
+
+    if (
```

Comment:
I would move this "isExpired" function into a `utils.ts` file so we can test it individually